packages/react-composites/src/composites/MeetingComposite/adapter/parseTeamsUrl.test.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getChatThreadFromTeamsLink } from './parseTeamsUrl';
+
+describe('Get chat thread from teams link', () => {
+  test('Should extract chat thread ID from teams url', async () => {
+    expect(
+      getChatThreadFromTeamsLink(
+        'https://teams.microsoft.com/l/meetup-join/19:meeting_ZTgzM2VmZWEtMXExMC00Yzk3LWE5NjUtYzg5ZDM3OTRjMzk0@thread.v2/0?context=%7B%22Tid%22:%2272f108bf-86f1-41af-91ab-2d7cd011db47%22,%22Oid%22:%222708ac84-aa11-4fab-8d13-e8586a26f6cd%22%7D'
```

Comment:
So... this is an active meeting that you're going to keep running, and the first 100 people to join enter a raffle draw?

Reply:
Hahah, I randomly changed a bunch of characters in this